### PR TITLE
Add coalition lead and site coordinator roles

### PR DESCRIPTION
**Pull request by jenny-heath**

### CODE REVIEW

File: app/models/admin_role.rb [hunk 1]
```diff
@@ -7,4 +7,6 @@
 #  updated_at :datetime         not null
 #
 class AdminRole < ApplicationRecord
+  TYPE = "AdminRole"
+  DISPLAY_NAME = I18n.t("general.admin")
```
Comment (alaroia-cfa) [inline app/models/admin_role.rb:11]: I believe this implementation approach computes the translation string at module load time, which means it'll always be English. I could be wrong, but consider checking that.

**Comment by alaroia-cfa** [comment]
I'm excited to see more roles 🥖

File: app/models/admin_role.rb (hunk 1 above)
Comment (jenny-heath) [reply app/models/admin_role.rb:11]: ah i checked and this is true - i will merge this as is but will leave a note in the story

**Comment by alaroia-cfa** [comment]
Works for me!

**Pull request merged**